simulation_ws/src/cloudwatch_simulation/launch/bookstore_turtlebot_navigation.launch.py

```diff
@@ -8,9 +8,6 @@
 TURTLEBOT3_MODEL = os.environ.get('TURTLEBOT3_MODEL', 'burger')
```

Comment:
this might not be required now

Reply:
good catch, will remove it

Reply:
actually it still needs the model? see
https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/blob/ros2/simulation_ws/src/cloudwatch_simulation/launch/bookstore_turtlebot_navigation.launch.py#L78